### PR TITLE
Require graphite for making lancers (no longer includes arcs)

### PR DESCRIPTION
**Pull request by Bohaska**
Nothing more needed

### CODE REVIEW

**Review by Quezler — APPROVED** [review]

File: core/src/mindustry/content/Blocks.java [hunk 1]
```diff
@@ -1514,7 +1514,7 @@ public void drawBase(Tile tile){}
         }};
 
         lancer = new PowerTurret("lancer"){{
-            requirements(Category.turret, with(Items.copper, 25, Items.lead, 50, Items.silicon, 45));
+            requirements(Category.turret, with(Items.copper, 25, Items.lead, 50, Items.silicon, 45 Items.graphite, 30));
```
Comment (Quezler) [inline core/src/mindustry/content/Blocks.java:1517]:
```suggestion
            requirements(Category.turret, with(Items.copper, 25, Items.lead, 50, Items.silicon, 45, Items.graphite, 30));
```

**Review by Quezler — CHANGES_REQUESTED** [review]

**Comment by GreyMario** [comment]
Uh... wait, _why,_ exactly? Is this because you got your ass whipped in PVP by Arcs and Lancers by sending ground units? 👀

**Comment by GreyMario** [comment]
Oh.

Found why this PR exists.

It's being submitted by proxy from _that_ guy ([1](https://github.com/Anuken/Mindustry-Suggestions/issues/1402), [2](https://github.com/Anuken/Mindustry-Suggestions/issues/1400)). The one who wants "balance changes" (read: nerfing things so hard that they effectively become banned in practice) because he can't deal with them in PVP.

**Comment by BlueWolf3682** [comment]
making these turrets cost graphite makes sense only for lancer, as any good player can use lancer a little *too* effectively against ground swarms. also @GreyMario these people have valid points, and theres absolutely no reason to be toxic to them. if youre wondering, ive played all gamemodes and i find them to be pretty much right with their actions. a lot of things need rebalancing.
take any arguments with me to discord. i dont want to litter the comments with peoples tantrums.

**Comment by GreyMario** [comment]
Sorry, but "I can't figure out how to send an air unit to go deal with the Lancer" isn't a valid reason to nerf the building with obscene build costs.

**Comment by Bohaska** [comment]
> Sorry, but "I can't figure out how to send an air unit to go deal with the Lancer" isn't a valid reason to nerf the building with obscene build costs.

I do not play pvp

**Comment by Bohaska** [comment]
> making these turrets cost graphite makes sense only for lancer, as any good player can use lancer a little _too_ effectively against ground swarms. also @GreyMario these people have valid points, and theres absolutely no reason to be toxic to them. if youre wondering, ive played all gamemodes and i find them to be pretty much right with their actions. a lot of things need rebalancing.
> take any arguments with me to discord. i dont want to litter the comments with peoples tantrums.

Ok.

**Comment by Bohaska** [comment]
Guys, are you all dead?

**Review by Quezler — APPROVED** [review]

**Comment by Bohaska** [comment]
@Anuken this is ready for review

**Comment by BlueWolf3682** [comment]
how is the build cost of lancer obscene? copper, lead, and silicon, all below 75 is extremely easy to build. if you cant get that much stuff, then actually make production.

**Comment by Anuken** [comment]
I don't think graphite should be involved. Increasing lead/copper requirements for longer build time should be enough.

**Pull request closed without merging**